src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -113,13 +114,15 @@ static void uct_rdmacm_cm_handle_event_route_resolved(struct rdma_cm_event *even
         return;
     }
 
-    ucs_trace("%s: rdma_connect, cm_id %p",
-              uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN), cep->id);
+    ucs_trace("%s: rdma_connect, cm_id %p, from address %s",
+              uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN), cep->id,
```

Comment:
shouldn't it be "to address" instead of "from address"?

Reply:
we are printing the address of the client that does 'rdma_connect' with rdma_get_**local**_addr so it's the address from which the connect request is sent to the server